tests/support/sh_py_run_test.bzl

```diff
@@ -13,14 +13,88 @@
 # limitations under the License.
 """Run a py_binary with altered config settings in an sh_test.
 
-This facilitates verify running binaries with different outer environmental
-settings and verifying their output without the overhead of a bazel-in-bazel
-integration test.
+This facilitates verify running binaries with different configuration settings
```

Comment:
This verbiage is a little odd to me. Is `verify` a command / action? If so, maybe add quotes or a reference?

```
This facilitates "verify" (the action) running binaries with different configuration settings
```

Reply:
Ah, that's a typo. It should say something more like

> This facilitates running binaries ... and verifying their output ...